ui/widgets/controlgroup.js

```diff
@@ -117,15 +117,19 @@ return $.widget( "ui.controlgroup", {
 				.each( function() {
 					var element = $( this );
 					var instance = element[ widget ]( "instance" );
+					var defaultInstance = instance || ( $.fn[ widget ].widgetConstructor &&
+							$.fn[ widget ].widgetConstructor.prototype );
+					var instanceOptions = $.widget.extend( {}, options );
 
 					// If the button is the child of a spinner ignore it
 					if ( widget === "button" && element.parent( ".ui-spinner" ).length ) {
 						return;
 					}
-					if ( instance ) {
-						options.classes = that._resolveClassesValues( options.classes, instance );
+					if ( defaultInstance ) {
```

Comment:
OK, instantiate with default options first, and then call options. I can do that.